pyccel/parser/semantic.py

```diff
@@ -853,8 +859,44 @@ def _visit_CommentBlock(self, expr, **settings):
         return expr
     def _visit_AnnotatedComment(self, expr, **settings):
         return expr
+
     def _visit_OmpAnnotatedComment(self, expr, **settings):
+        code = expr._user_nodes
```

Comment:
I thought codacy usually complained about this kind of statement. Is there no public method letting you get this?

Reply:
i don't think there a method for this